### ISSUE TITLE
Masked Gradient-Based Causal Structure Learning

### ISSUE THREAD

**Issue by hiremasa** [hiremasa/Causal_Discovery_Paper_Summary#70]
## 一言でいうと
A gradient-based optimization framework; Masaked Gradient-based Causal Structure Learning (MGCSL)
1) flexible includes different model functions (NN, poly, ...)
2) is easily thresholded to identify the edges

### 論文リンク
- [SDM](https://epubs.siam.org/doi/10.1137/1.9781611977172.48)
- [Implementation]()
## 新規性
To achieve their goals, they reformulate the SEM with additive noises in a form parameterized by binary adjacency matrix that exists for any data model and is called mask in the paper — the binary-valued entry can be used to remove the effect of a variable on the output of any function.

To enable efficient learning of binary-valued entries, they leverage the Gumbel-Softmax approach and develop a gradient-based structure learning method with a carefully devised training procedure.
